internal/contributebot/design/README.md

```diff
@@ -1,4 +1,4 @@
-# Go Cloud Contribute Bot Design
+# Go Cloud Development Kit Contribute Bot Design
```

Comment:
Here and below (esp. in the code/config files) - should we just use "Go CDK" for all the internal stuff? It's less typing, and presumably anyone reaching this deep already knows what CDK means.  In particular, "Go Cloud Development Kit Contribute Bot Design" is quite a mouthful :)

Reply:
Done. In some places I just removed it entirely. I did leave the full name in some code files, e.g., for the UI that the running Contribute Bot shows.